src/Delivery/ResourceBuilder.php

```diff
@@ -169,14 +169,17 @@ private function buildArray(array $data, array $rawDataList = null)
      */
     private function buildAsset(array $data)
     {
+        $sys = $this->buildSystemProperties($data['sys']);
+        $locale = $sys->getLocale();
+
         $fields = $data['fields'];
-        $files = array_map([$this, 'buildFile'], $fields['file']);
+        $files = array_map([$this, 'buildFile'], $this->normalizeFieldData($fields['file'], $locale));
 
         $asset = new Asset(
-            isset($fields['title']) ? $fields['title'] : null,
-            isset($fields['description']) ? $fields['description'] : null,
+            isset($fields['title']) ? $this->normalizeFieldData($fields['title'], $locale) : null,
+            isset($fields['description']) ? $this->normalizeFieldData($fields['description'], $locale) : null,
             $files,
-            $this->buildSystemProperties($data['sys'])
+            $sys
```

Comment:
So if I'm understanding this correctly, `buildAsset` returns an `Asset` object that only reflects the data of one locale (e.g. it has been localized)?